libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -93,6 +93,8 @@ class AP_Motors {
         uint8_t yaw             : 1; // we have reached yaw limit
         uint8_t throttle_lower  : 1; // we have reached throttle's lower limit
         uint8_t throttle_upper  : 1; // we have reached throttle's upper limit
+        uint8_t motor_lower     : AP_MOTORS_MAX_NUM_MOTORS; // a certain motor has reached the lower warning threshold
```

Comment:
No, AP_MOTORS_MAX_NUM_MOTORS is just the width of the field in bits, as the new field is a  bitmask (obviously the same width for both upper and lower limit).